crates/sound/src/audio_backends/pw_backend.rs

```diff
@@ -2,13 +2,16 @@
 // SPDX-License-Identifier: Apache-2.0 or BSD-3-Clause
 
 use super::AudioBackend;
-use std::{thread};
-use std::{cell::Cell, rc::Rc};
+use std::ptr;
+use std::ops::{Deref};
+use std::ptr::NonNull;
 use crate::Result;
 
 use vm_memory::Le32;
 use pipewire as pw;
-use pw::sys::PW_ID_CORE;
+use pw::sys::{pw_loop, PW_ID_CORE};
+use pw::sys::*;
```

Comment:
This should include all the symbols of the previous line.
Now that it's just a PoC it's okay, but in the future maybe better to just add the symbols we use.

Reply:
i thought about it, but then realised they were too much and just did pw::sys::* i can specifically add only the symbols we use.

Reply:
I agree with @stefano-garzarella here, it is a bad practice to import everything unless it is for prelude traits or enum members. You can call `pw::sys::some_ffi_func` directly. It is short, and give you a direct justification of why unsafe is needed.